<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QueryStu</class>
 <widget class="QDialog" name="QueryStu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>查询学生信息</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,9">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,6,2">
     <item>
      <widget class="QComboBox" name="cbb_method">
       <property name="font">
        <font>
         <family>黑体</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <item>
        <property name="text">
         <string>请选择查询方式</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>按姓名查询</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>按学号查询</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>按性别查询</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>按年龄查询</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>按院系查询</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_search">
       <property name="font">
        <font>
         <family>黑体</family>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_search">
       <property name="font">
        <font>
         <family>黑体</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Search</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="tableView">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>9</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 9pt &quot;黑体&quot;;
background-color: rgb(237, 254, 255);</string>
     </property>
     <property name="gridStyle">
      <enum>Qt::NoPen</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
